### DISCUSSION TITLE
Record video from both front & rear cam at the same time

### DISCUSSION THREAD

**Discussion by JasPanesar** [Apparence-io/CamerAwesome#194] [Q&A]
.

**Comment by istornz** [comment]
Hello @JasPanesar, for the moment it's not feasible with the version 1.0.0 of camerAwesome.
We planned to add this feature later !

**Reply by JasPanesar** [reply]
Sounds good I'm going to keep an eye out for this as well.  I think there was another camera library but it's lacking different features.
